### DISCUSSION TITLE
Subtitle Edit - working with Burnt In Timecode

### DISCUSSION THREAD

**Discussion by jazmintee** [SubtitleEdit/subtitleedit#5850] [Q&A]
Hello,  I'm new to Subtitle Edit and am wanting to use the software to create Audio Descriptions.
The video clips I work with have BITC so it's absolutely imperative that this timecode is calibrated and matches the onscreen software timecodes but it isn't.  Is there any way of doing this please?
For instance my client's video with burnt in timecode starts at 00:58:00:00 but SE begins at 00:00:00:00 so the Audio Description
in and out points will not match.  Is there anyway to fix this, as I am unable to find anything.
Thanks!

**Comment by darnn** [comment]
Try video -> set video offset. Note that there's also video -> SMPTE timing, if that's what your files use.

**Reply by jazmintee** [reply]
Thank you that's a great help! I'm not very techy unfortunately - but I think I've worked it out now.
